docs/routing.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Blazor routing
+author: guardrex
+description: Learn how to route requests in a client-side Blazor app and about the NavLink component.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 05/01/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/routing
+---
+# Blazor routing
+
+By [Luke Latham](https://github.com/guardrex)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+Learn how to route requests in a client-side Blazor app and about the NavLink component.
+
+[View or download sample code](https://github.com/aspnet/Blazor.Docs/tree/master/docs/components/common/samples/) ([how to download](xref:client-side/blazor/index#view-and-download-samples)). See the [Get started](xref:client-side/blazor/get-started) topic for prerequisites.
+
+## Route templates
+
+Routing in Blazor is achieved by providing a route template to each accessible component in the app.
```

Comment:
Routing only works if you first add the Router component to your app. In our templates this is done in *App.cshtml*

```html
<Router AppAssembly=typeof(Program).Assembly />
```

Setting up the Router component will probably move to *Program.cs* some day, but for now this is how it's done.